src/public/styles.css

```diff
@@ -80,3 +108,44 @@ input:checked + label {
 input {
   display: block;
 }
+
+/* signup.html */
+
+header {
```

Comment:
if you have multiple pages with multiple headers you might want to style them all differently. I'd definitely look into learning about [BEM](https://css-tricks.com/bem-101/) when you get time. Lot's of people use it, but I think even more people use an approximation of it... just take the philosophy and apply it how you see fit kind of thing.